docs/csharp/language-reference/compiler-options/language.md

```diff
@@ -23,13 +23,13 @@ The following options control how the compiler interprets language features. The
 
 ## CheckForOverflowUnderflow
 
-The **CheckForOverflowUnderflow** option specifies whether an integer arithmetic statement that results in a value that is outside the range of the data type causes a run-time exception.  
+The **CheckForOverflowUnderflow** option specifies whether an integer arithmetic expression that results in a value that is outside the range of the destination type causes a run-time exception.  
```

Comment:
Same comment.

Reply:
However, here we should mention user-defined checked operators and possible no-exception case. I'll think about how to do that.

Reply:
@BillWagner at the end, I've rewritten this section. Please check my latest `Addressed feedback` commit

Reply:
I had one minor grammar nit, and then this is ready.

Thanks again @pkulikov